genet/inputs_handler/osm_reader.py

```diff
@@ -116,16 +116,16 @@ def create_s2_indexed_osm_graph(response_jsons, config, num_processes, bidirecti
 
 
 def generate_graph_nodes(nodes, epsg):
-    input_to_output_transformer = Transformer.from_crs('epsg:4326', epsg)
+    input_to_output_transformer = Transformer.from_crs('epsg:4326', epsg, always_xy=True)
```

Comment:
Would we ever want `always_xy=False` (which is the default, I guess)? Wondering if we should create our own wrapper around this in the form of a factory function or suchlike.

Reply:
No idea. I tried to see why you would ever set this as False or feel the need to swap the order of points and I just cannot understand it.